cmd/agent/app/config.go

```diff
@@ -78,3 +97,65 @@ func requestConfig() (string, error) {
 
 	return string(r), nil
 }
+
+func listRuntimeConfigurableValue(cmd *cobra.Command, args []string) error {
+	err := util.SetAuthToken()
+	if err != nil {
+		return err
+	}
+	c := util.GetClient(false)
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return err
+	}
+	url := fmt.Sprintf("https://%v:%v/agent/config/list-runtime", ipcAddress, config.Datadog.GetInt("cmd_port"))
+	r, err := util.DoGet(c, url)
+	if err != nil {
+		var errMap = make(map[string]string)
+		json.Unmarshal(r, &errMap)
+		// If the error has been marshalled into a json object, check it and return it properly
+		if e, found := errMap["error"]; found {
+			return fmt.Errorf(e)
+		}
+		return err
+	}
+	var settings = make(map[string]string)
+	err = json.Unmarshal(r, &settings)
+	if err != nil {
+		return err
+	}
+	fmt.Println("=== Settings that can be changed at runtime ===")
+	for setting, desc := range settings {
+		fmt.Printf("%s: %s\n", setting, desc)
```

Comment:
What do you think of using something like: `%s\t\t\t%s\n`. Feel free to ignore.

Reply:
I agree, larger space look nicer.